common-api/src/main/java/org/pmiops/workbench/cdr/dao/DomainInfoDao.java

```diff
@@ -0,0 +1,51 @@
+package org.pmiops.workbench.cdr.dao;
+
+import java.util.List;
+import org.pmiops.workbench.cdr.model.DomainInfo;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+public interface DomainInfoDao extends CrudRepository<DomainInfo, Long> {
+
+  // TODO: consider not using @Query since it doesn't let us re-use shared SQL expressions for these methods?
+
+  /**
+   * Returns domain metadata and concept counts for domains, matching standard concepts by name
```

Comment:
Possibly existing issue, but I can't tell what the parameters are controlling given the vague naming, could you add docs?

Reply:
I updated the parameter name and docs here and in SurveyModuleDao.